lib/exception_handling/exception_info.rb

```diff
@@ -78,6 +78,10 @@ def honeybadger_context_data
       @honeybadger_context_data ||= enhanced_data_to_honeybadger_context
     end
 
+    def controller_name
+      @controller_name ||= (@controller ? @controller.request.parameters.with_indifferent_access[:controller] : nil).to_s
```

Comment:
Personally, I'm not a fan of explicit nil returns, but up to you.
```
@controller_name ||= if @controller 
                      @controller.request.parameters.with_indifferent_access[:controller]
                     end.to_s

Reply:
Oh, yeah I like that better. Will update.